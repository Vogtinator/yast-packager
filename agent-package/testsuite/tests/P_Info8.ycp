{
return Execute(.setEnvironment,$["packageinfopath":"tests","language":"cs_CZ","common.pkd":"common.ori.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return Read(.selGroups);
}
{
    return Execute(.selectSelInstall, "default", true );
}
{
    return Read(.selInstallSet);
}
{
    return Read(.selDependencies);
}
{
    return Read(.selPackages);
}
