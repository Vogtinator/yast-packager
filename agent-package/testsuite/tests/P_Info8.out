(true)
([["Devel", " V√Ωvoj", "addon"], ["Games", " Hry", "addon"], ["Gnome", " Syst√©m Gnome", "addon"], ["Kde", " V≈°e z KDE", "addon"], ["Kde-Desktop", " KDE Desktop Environment", "addon"], ["LAMP", " Simple Webserver", "addon"], ["Min", " Minimum system", "baseconf"], ["Minimal", " Minim√°ln√≠ syst√©m", "baseconf"], ["Minimal+X11", " Minim√°ln√≠ grafick√Ω syst√©m (bez KDE) ", "baseconf"], ["Multimed", " Multim√©dia", "addon"], ["Multimedia", " Multim√©dia", "addon"], ["Network", " S√≠≈•/Server", "addon"], ["Standard", " Standardn√≠ syst√©m", "baseconf"], ["default", " Syst√©m s kancel√°≈\231sk√Ωm bal√≠kem", "baseconf"], ["xxxx", " xxxx", "baseconf"], ["zzz_Alles", " V≈°echny bal√≠ky (trv√° velmi dlouho)", "baseconf"]])
(true)
(["Kde-Desktop", "default"])
($["ADD":[], "CONFLICT":[], "REQUIRE":[]])
(["3ddiag", "ImageMagick", "RealPlayer", "SuSEfirewall", "WindowMaker", "a2ps", "aaa_base", "aaa_dir", "aaa_skel", "acroread", "alsa", "alsa-devel", "apmd", "aps", "ash", "at", "audiofile", "autofs", "awesfx", "base", "bash", "bc", "bdflush", "bin86", "bindutil", "binutils", "bzip", "calctool", "cdparanoia", "compat", "compress", "cpio", "cpp", "cracklib", "cron", "ctags", "cyrus-sasl", "db", "devs", "dhcpcd", "dialog", "diffutils", "dosfstools", "e2fsprogs", "ed", "eject", "emacs", "emacs-info", "emacs-x11", "esound", "fam", "fbset", "fetchmail", "file", "fileutils", "filters", "findutils", "finger", "finger-server", "freetype", "freetype2", "ftpd", "fvwm2", "g3utils", "gawk", "gcc", "gdbm", "gfxboot", "glib", "glibc", "glibc-devel", "gpg", "gpm", "gpp", "gppshare", "grep", "groff", "gtk", "gv", "gzip", "hdparm", "heimdal-lib", "hermes", "html2ps", "hwinfo", "i4l", "i4lfirm", "iamerica", "icons", "icons-xpm", "iczech", "idedma", "ifnteuro", "imlib", "imwheel", "inetd", "intlfnts", "ipchains", "iproute2", "iputils", "isapnp", "isax", "ispell", "java", "joe", "joystick", "jpeg", "kbd", "kde-i18n", "kde-i18n-cs", "kdeadmin", "kdebase", "kdebase-ksysguardd", "kdebase-nsplugin", "kdebase-yast2", "kdegames", "kdegraphics", "kdelibs", "kdelibs-devel", "kdemultimedia", "kdenetwork", "kdetoys", "kdeutils", "kdialog", "kfontinst", "kinternet", "less", "libPropList", "libgpp", "libjpeg", "liblcms", "libmikmo", "libmng", "libnetpbm", "libogg", "libpng", "libscr", "libsmi", "libstroke", "libtiff", "libungif", "libvorbis", "libxml", "libxml2", "liby2", "libycp", "libyui", "libz", "lilo", "listexec", "lprng", "lsof", "lukemftp", "lvm", "lynx", "m4", "mailx", "make", "makewhat", "man", "man-pages", "mc", "mesa", "mesaglut", "mesasoft", "metamail", "mktemp", "modutils", "mtools", "mutt", "ncurses", "ncurses-devel", "net-tools", "netcat", "netcfg", "netdate", "netpbm", "netscape", "nfs-utils", "openldap2-client", "openmotif", "openssh", "openssl", "pam", "pam_devperm", "patch", "pciutils", "pcmcia", "pcre", "perl", "perl-Storable", "perl-Tk", "perl-gettext", "personal-firewall", "pico", "pine", "popt", "portmap", "ppp", "procmail", "providers", "ps", "qt", "qt-devel", "raidtools", "rawio", "readline", "recode", "reiserfs", "rpm", "rsh", "rsh-server", "sash", "sax", "sax2", "saxtools", "screen", "sendmail", "setserial", "sh-utils", "shadow", "sharutils", "siga", "slang", "smbclnt", "so_base", "so_en", "strace", "sudo", "susehelpcenter", "susewm", "syslogd", "sysvinit", "talk", "talk-server", "tcl", "tcpd", "tcpdump", "tcsh", "telnet", "telnet-server", "terminfo", "texinfo", "textutils", "timezone", "tin", "tix", "tk", "traceroute", "transfig", "ttmkfdir", "tuxeyes", "unclutter", "ungif", "unrar", "unzip", "usbview", "util-linux", "vim", "vlna", "w3m", "wget", "wmapps", "wmthemes", "words", "wterm", "wvdial", "xanim", "xaw3d", "xbanner", "xcolors", "xdevel", "xdmbgrd", "xf86", "xf86_3x", "xf86glu", "xf86tools", "xfine", "xfnt100", "xfntl2", "xfntscl", "xkeycaps", "xless", "xloader", "xlock", "xman", "xmms", "xmms-kde", "xmods_3x", "xmodules", "xntp", "xosview", "xshared", "xtermset", "xvga16", "yacc", "yast", "yast2", "yast2-agent-any", "yast2-agent-cups", "yast2-agent-fdisk", "yast2-agent-isax", "yast2-agent-modules", "yast2-agent-probe", "yast2-agent-rcconfig", "yast2-base", "yast2-config-adsl", "yast2-config-bootfloppy", "yast2-config-bootloader", "yast2-config-cups", "yast2-config-environment", "yast2-config-hwinfo", "yast2-config-inet", "yast2-config-network", "yast2-config-online-update", "yast2-config-package-manager", "yast2-config-printer", "yast2-config-rcconfig", "yast2-config-security", "yast2-config-sound", "yast2-config-tune", "yast2-config-update", "yast2-config-users", "yast2-core-clients", "yast2-core-pkginfo", "yast2-core-scr", "yast2-core-wfm", "yast2-db-groups", "yast2-db-keyboard", "yast2-db-mouse", "yast2-db-printer", "yast2-db-sound", "yast2-db-timezone", "yast2-lib-printer", "yast2-lib-sequencer", "yast2-lib-wizard", "yast2-menu", "yast2-module-support", "yast2-trans-adsl", "yast2-trans-boot", "yast2-trans-bootfloppy", "yast2-trans-cups", "yast2-trans-hwinfo", "yast2-trans-inet", "yast2-trans-inst-bootloader", "yast2-trans-inst-environment", "yast2-trans-inst-general", "yast2-trans-inst-language", "yast2-trans-inst-packages", "yast2-trans-inst-partitioning", "yast2-trans-inst-update", "yast2-trans-inst-user", "yast2-trans-inst-x11", "yast2-trans-libprinter", "yast2-trans-menu", "yast2-trans-network", "yast2-trans-online-update", "yast2-trans-package-manager", "yast2-trans-printer", "yast2-trans-rcconfig", "yast2-trans-security", "yast2-trans-sound", "yast2-trans-support", "yast2-trans-tune", "yast2-trans-update", "yast2-trans-users", "yast2-trans-wizard", "yast2-ui-ncurses", "yast2-ui-qt", "yp-tools", "ypbind", "zip", "ziptool"])
